### ISSUE TITLE
Spiritual tab

### ISSUE THREAD

**Issue by Enderborn1235** [Azanor/thaumcraft-suggestions#109]
I had another idea after replying to one of webwraith's ideas about using wisps as golem brains -- what if there was a way to get into a new tab after scanning wisps. Thus the incorporeal tab idea was formed. The idea got its inspiration from the mod Intangible and Thaumcraft's totems and wisps. Some of the ideas are as follows:

Incorporeal form: Everything living eventually dies. We all know this. But what we didn't know is that spirits exist beyond death, avenging their own deaths. With this you have devised a way to tear the spirits from the living without killing them. You must make a multiblock structure made of totem base blocks, followed by a spiritual matrix(runic matrix infused with balance shards and ender pearls with dangerous instability). However, this way is imperfect as the shells have a tendency to rot after a day or two. Only living creatures can be torn of their souls, as undead don't have souls. As fuel, it also takes spiritus essentia.

Spiritometer: you have one issue when you perform the ritual... How can you see the spirits? By infusing a thaumometer with spiritus(you can also make a goggles of revealing version), you can crudely make out the shape of the spirits to observe.

Spiritual Desires: You have noticed something quited irregular. Depending on how you treat the spirit's body when it is alive, it will return the favor when it is dead. Let us say your favored pet wolf or ocelot dies, it will have the desire to be with you until you die. The unfortunate side is that it is weak, lacking in power. Now let's say your least favored pet dies. It will be powerful and a sight to see, but the catch is it will be extremly hostile towards you for treating it as such and will try to destroy all humans within its wake

I am open to other ideas if you have them. I will update this issue with more ideas when I get them.

Thanks,

-Enderborn1235

Edit: Minor typo fix on some words
Edit 2: Removed spiritual alignment and infusion, and added spirtual desires

**Comment by Enderborn1235** [comment]
I would like to note that this is the basic idea which, can and will, be extended.

**Comment by cadaverous-eris** [comment]
...but what would this accomplish that current infusion crafting can't? This just seems like it would make you need to move your altar after crafting a certain amount of things.

**Comment by Enderborn1235** [comment]
The infusion idea is more like infusion enchanting with more random abilities base on the vis and spirits in the area

**Comment by cadaverous-eris** [comment]
But the whole point of infusion enchants are that you know exactly what you're getting. If you want to add random enchants, it makes more sense to just use the vanilla enchanting mechanic. Otherwise, it would only be worth it if you could get extra powerful enchantments from it.
...Actually, that would be kinda cool.
But never the less, the whole spirit system seems like it isn't entirely necessary, and really just seems like it's on,y purpose would be to circumnavigate the aura unification in TC6, at which point it would make more sense to just separate the aura types again.

**Comment by Enderborn1235** [comment]
True, I will have to update the idea accordingly. The gist is spirits exist. To use them or avoid them is your choice

**Comment by webwraith** [comment]
Perhaps this could allow very powerful enchants, but the more times you use it, the more powerful spirits come after you, akin to the warp event-caused Guardian spawn, but getting more armour and more powerful attacks with each spirit bound. It would act like permanent warp, so you can only mitigate the attacks, and not get rid of them entirely. Perhaps the enchantments actually refuse to work while the player is in battle against the avenging spirits?

**Comment by Enderborn1235** [comment]
Thanks for the idea, as this whole idea sort of stemmed from one of your ideas
